deeprank/tools/pdb2sql.py

```diff
@@ -113,7 +113,8 @@ def _create_sql(self):
                     'y': 'REAL',
                     'z': 'REAL',
                     'occ': 'REAL',
-                    'temp': 'REAL'
+                    'temp': 'REAL',
+                    'element': 'TEXT'
```

Comment:
I don't know what this element is but we should also use it in pdb2sql then ... and also use pdb2sql in deeprank :)

Reply:
The element here is chemical element.

Reply:
using pdb2sql will maybe not work out of the box as some functionalities ma be slightly different ... so be careful there